src/pages/transactions/messages.tsx

```diff
@@ -0,0 +1,42 @@
+import Head from 'next/head'
+import { Typography, Link, SvgIcon } from '@mui/material'
+import type { NextPage } from 'next'
+
+import LinkIcon from '@/public/images/common/link.svg'
+import NoMessagesIcon from '@/public/images/messages/no-messages.svg'
+import PagePlaceholder from '@/components/common/PagePlaceholder'
+import TxHeader from '@/components/transactions/TxHeader'
+import NavTabs from '@/components/common/NavTabs'
+import { transactionNavItems } from '@/components/sidebar/SidebarNavigation/config'
+
+const History: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>Safe – Messages</title>
+      </Head>
+
+      <TxHeader action={<NavTabs tabs={transactionNavItems} />} />
+
+      <main>
+        <PagePlaceholder
+          img={<NoMessagesIcon />}
+          text={
+            <Typography variant="body1" color="primary.light" m={2} maxWidth="600px">
```

Comment:
`variant="body1"` is the default. No need to specify IIRC

Reply:
mockup has `font-size: 14px`

Reply:
I've removed the prop.

With regards to the `font-size`, this placeholder "styling" mirrors that of the [`SafeAppsSearchPlacholder`](https://github.com/safe-global/web-core/blob/dev/src/components/safe-apps/SafeAppsSearchPlaceholder.tsx).

Should we alter that or keep this as is so that they match? cc @liliiaorlenko

Reply:
non-blocking. will resolve and can be adjusted later